images/scripts/rhel.setup

```diff
@@ -296,6 +289,14 @@ EOF
     version=$(cut -d '-' -f 2 <<< "$IMAGE")
     su builder -c "/usr/bin/mock --verbose -i $(/var/lib/testvm/build-deps.sh "$opsys $version")"
     su builder -c "/usr/bin/mock --install --verbose rpmlint"
+
+    # We need to make sure mock has the same selinux-policy as we do
+    if [ "${IMAGE#rhel-9*}" != "$IMAGE" ]; then
+        dnf update -y selinux-policy
+        our_selinux_policy_version="$(rpm -q --qf '%{EVR}' selinux-policy)"
+        selinux_mock_packages="selinux-policy{,-devel,-targeted}-${our_selinux_policy_version}"
+        su builder -c "/usr/bin/mock --addrepo='$archive_repo' --install ${selinux_mock_packages}"
```

Comment:
`archive_repo` is not set, afaics.

Reply:
Ya.  That was a mistake.  Trying a different approach anyway.